src/Stratis.Bitcoin.Features.SmartContracts.Tests/Lifecycle/SmartContractRestorerTests.cs

```diff
@@ -31,61 +31,65 @@ public SmartContractRestorerTests()
         [Fact]
         public void SmartContract_Restorer_NonpublicStateFieldsSetSuccess()
```

Comment:
@noescape00 can you push master to your merge branch again? These changes shouldnt be here. I think this file was deleted.

Reply:
Sure
Fixed